astroquery/alfalfa/tests/test_alfalfa.py

```diff
@@ -38,7 +38,9 @@ def patch_get_readable_fileobj(request):
     @contextmanager
     def get_readable_fileobj_mockreturn(filename, **kwargs):
         file_obj = data_path(DATA_FILES['spectrum'])  # TODO: add images option
-        yield open(file_obj, 'rb')  # read as bytes, assuming FITS
+        # read as bytes, assuming FITS
+        with open(file_obj, 'rb') as inputfile:
+            yield inputfile
```

Comment:
`Path` has a `read_bytes()` method: https://github.com/python/cpython/blob/feb4be1553c947ddb2bd8307419451bfdfc0a6a3/Lib/pathlib.py#L1122-L1127

Reply:
I know, but tags a reactor for another day